data/site-list.json

```diff
@@ -60,7 +60,7 @@
       "tilt": null,
       "latitude": 50.0,
       "longitude": 50.0,
-      "installed_capacity_kw": 4,
+      "installed_capacity_kw": 0,
```

Comment:
wait why did these get changed to 0?

Reply:
This is because this sites never load a forecast since there's no data for them, and to avoid confusion in the way the aggregate site capacity is calculated, being that it sums the several sites, I set these to 0. That way, the threshold, like the forecast values, both stay the same from the aggregate to single site dashboard (for the one that works)